utils/utils.js

```diff
@@ -0,0 +1,36 @@
+const request = require('request');
+const errorPhrases = require('../assets/errorPhrases.json'); 
+
+asyncRequest = (url) => {
+    return new Promise( (resolve, reject) => {
+        request(url, (error, res, body) => {
+            if (!error && (res.statusCode == 200 || res.statusCode == 301)) {
+                resolve(body);
+            } else {
+                reject(error);
+            }
+        });
+    });
+};
+
+
+logAndMsg = (channel, msg) => {
```

Comment:
We don't actually us this function right now, but I suspect we might in the future.